src/main/kotlin/traffic_simulation/Vehicle.kt

```diff
@@ -16,8 +17,8 @@ class Vehicle(val id: Int, val wannaDriveInHours: MutableList<Int>) {
         // (otherwise it just waited an hour, not reducing its delay)
         if (wannaDriveInHours.contains(timestep)) {
             delay += 1
-            this.delayedInHours.add(timestep)
-            this.delayedInHours.sort()
+            this.gotNewDelayInHours.add(timestep)
```

Comment:
Ja, das ist Absicht. Finde das so konsistent mit unserer Philosophie von delay.

Im Output wird ausgegeben, wann ein Fahrzeug eine zusätzliche Verspätung bekommt. Daher kommt das Adden zu der Liste in die if-Schleife.